interchain-exchange/tutorial/02-app-init.md

```diff
@@ -36,25 +36,23 @@ Scaffold a module inside your blockchain named `ibcdex` with IBC capabilities.
 The ibcdex module contains the logic for creating and maintaining order books and routing them through IBC to the second blockchain.
 
 ```bash
-starport module create ibcdex --ibc --ordering unordered
+starport scaffold module ibcdex --ibc --ordering unordered
 ```
 
 ## Create the Transaction Types
 
-To scaffold two types with create, read, update and delete (CRUD) actions use the Starport `type` command.
-The following commands create `sellOrderBook` and `buyOrderBook` types. 
+To scaffold two types with create, read, update and delete (CRUD) actions use the Starport `map` command.
```

Comment:
```suggestion
To scaffold two types with create, read, update, and delete (CRUD) actions use the Starport `map` command.
```

Reply:
our style is to use the serial comma per https://developers.google.com/style/commas 
@lubtd I could talk about grammar all day long!